docs/Objectives/Objectives-and-Goals.md

```diff
@@ -0,0 +1,104 @@
+# Objectives and Goals of a Hack Week
+
+Key to the hack week organization process--and perhaps the process of organizing any meeting---is 
+a clear understanding what the hack week is *for*. What goals and objectives are the organizers 
+pursuing by organizing this hack week? Are these goals *personal* goals or *community-level* goals? 
+Some community-level goals might include bringing more data science methods into the community, 
+building a cohort of like-minded researchers, making progress on a specific tool, software package 
+or data set, and building a set of tools to address one specific or multiple scientific challenges. 
+
+Our hack weeks are very non-uniform in their goals and objectives. For example, Astro Hack Week 
+pitches itself as widely as possible: while the focus of the tutorials is often on new methods and 
+tools, any broadly astronomy-themed hack is welcome. This could include developing new software, but 
+could also mean providing a document of resources for teaching, designing a new logo for an organization, 
+or building tutorials for computational and astronomy subjects. Geo Hack Week, in contrast is more narrow 
+in scope and focuses on improving computational and programming literacy in the Geosciences community, and 
+on working directly on software tools that address geophysical challenges. The IceSat-2 Hack Week is 
+even more narrowly scoped on developing next-generation tools for a new space mission. 
+
+It is worth pointing out explicitly that none of those is better than the other. They are designed 
+differently to solve different challenges in different communities, and the hack week model easily 
+accomodates different goals that the organizers might pursue. In any of these cases, however, it is 
+important to be clear among the organizers what those goals and objectives actually are, and perhaps 
+spell them out, for example in a mission statement. They form the basis of much of the subsequent 
+organization, ranging from questions of whether participants require pre-knowledge during participant 
+selection to facilitating the hacks themselves.
+
+Ideally, an organizing committee should start with the question: *What are we trying to achieve?*
+From there, one can write down a set of big-picture themes that provide the basis for more specific 
+goals, as well as actions that help achieve those goals. 
+In this context, it is also helpful for organizers to discuss and settle on a set of *core values* that 
+the organizers aim to follow in their organization of the workshop. These could be values like kindness, 
+collaborativeness, and attentiveness, to name a few. The values defined in this stage might in part form 
```

Comment:
Suggest something here to say that these values are built on the foundation of the core hackweek mission. The point here is that each event doesn't have to start from scratch, and that there should be a process whereby the individual hack week core values do not diverge from the overall hackweek principles. Maybe:
"These values will align with the overall mission of hack weeks and will represent the specific dimensions of that mission that the organizers wish to amplify."

Reply:
Updated with your suggestion :)